source/SH3/arc/vfile.cpp

```diff
@@ -23,13 +23,18 @@ bool sh3_arc_vfile::Open(sh3_arc& mft, const std::string& fname)
         it (so we know how large it is without probing) though most headers contain the size of the
         full file
     */
-    if((fsize = mft.LoadFile(fname, buffer)) == ARC_FILE_NOT_FOUND)
+    int size = mft.LoadFile(filename, buffer);
```

Comment:
It's a perfectly fine technique, however clang complains about some implicit casts, which I avoid by buffering the value in a local `int`.